recipes/grass-session/run_test.py

```diff
@@ -0,0 +1,4 @@
+try:
+    import grass_session
+except RuntimeError:
+    pass
```

Comment:
This is not a good test.  Can you please remove the try-except?  Where is this RuntimeError coming from?

Reply:
I know, but the basic import will search for a binary for which no conda package exists. This is what triggers the RuntimeError. See discussion above (https://github.com/conda-forge/staged-recipes/pull/11808#discussion_r434581359)

Reply:
Oh, then please package grass